rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -1,32 +1,22 @@
+#
+#  Copyright (C) 2011-2017 Greg Landrum
```

Comment:
A comment that has nothing to do with the PR: 

If you want to keep the Copyright statements always correctly dated, it would be useful to consider writing a script that goes through all the code and fixes the latest year once a year.